docs/cow-protocol/tutorials/cow-swap/twap.mdx

```diff
@@ -98,7 +86,7 @@ The "Order submitted successfully" screen indicates that your order has been ini
 
 <Image img={require('/img/cowswap/TWAP_orderSigning.png')} alt="Order in signing state" />
 
-The "Order in signing state" screen shows that your order may require additional signatures from your Safe. Ensure all necessary parties sign to proceed.
+The "Order in signing state" screen shows that your order may require additional signatures from your `Safe`. Ensure all necessary parties sign to proceed.
 <br />
 
 <Image img={require('/img/cowswap/TWAP_ordersOverview.png')} alt="Orders overview" />
```

Comment:
Can we have some text for context in between these images?
When I see this, I'm not sure what the tutorial is trying to show me 
![image](https://github.com/cowprotocol/docs-v2/assets/43217/416c3eef-8171-4ee4-b398-89c2bb6d0833)

Or at least indicate in the image what the user should look at.
In care there are multiple points of attention, the number-inside-red-dot from the parameters section is an amazing option.

Reply:
#248 Refers